pytket/qir/conversion/conversion.py

```diff
@@ -172,12 +173,23 @@ def __init__(
             ),
         )
 
-        # i64 reg2var(i1, i1, i1, ...)
-        self.reg2var = self.module.module.add_external_function(
-            "reg2var",
+        # i1* create_reg(i64 size)
+        self.create_reg = self.module.module.add_external_function(
+            "create_reg",
             pyqir.FunctionType(
-                pyqir.IntType(self.module.module.context, qir_int_type),
-                [pyqir.IntType(self.module.module.context, 1)] * qir_int_type,
+                self.qir_i1p_type,
+                [pyqir.IntType(self.module.module.context, qir_int_type)],
+            ),
+        )
+
+        # i64 create_reg(i1* reg)
```

Comment:
```suggestion
        # i64 get_int_from_creg(i1* creg)
```
Specifically `read_all_bits_from_reg` could be better called `get_int_from_creg`